seqr/views/apis/dashboard_api.py

```diff
@@ -40,10 +40,11 @@ def dashboard_page_data(request):
 
 
 def _get_projects_json(user):
-    projects = get_projects_user_can_view(user)
-    if not projects:
+    project_guids = get_project_guids_user_can_view(user)
+    if not project_guids:
         return {}
 
+    projects = Project.objects.filter(guid__in=project_guids)
```

Comment:
this is the change that actually fixes the out of memory error. Before, `projects` here in the anvil-only case was 
```
projects = Project.objects.filter(can_view_group__user=user)
...
projects = (projects | _get_analyst_projects())
...
anvil_permitted_projects = Project.objects.annotate(
            workspace = Concat('workspace_namespace', Value('/'), 'workspace_name')).filter(workspace__in=workspaces)
return (anvil_permitted_projects | projects).distinct()
```

calling annotate for all the counts on this query was causing everything to blow up. While this change technically can mean there are 2 queries occurring instead of only one (although frequently the first query will be cached), the huge annotation step is being called on a much smaller and better scoped query